include/libtorrent/config.hpp

```diff
@@ -93,9 +89,6 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #elif defined BOOST_MSVC
 
-#pragma warning(disable: 4258)
```

Comment:
do you know if this introduced any new warnings? simulations and the python binding isn't warnings-as-errors yet (but The currently building PR on appveyor changes that). But if this patch makes it in first, it would be nice to make sure this warning is issued.